fastlane/README.md

```
@@ -0,0 +1,56 @@
+fastlane documentation
+----
+
+# Installation
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)